assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -37,16 +38,44 @@ import Checkbox from '../Checkbox';
 import { Cell, Input, TextField } from '../../material-components';
 const { useSelect, useDispatch } = Data;
 
-export default function UserInputSelectOptions( { slug, options, max } ) {
+export default function UserInputSelectOptions( { slug, options, max, next, isActive } ) {
 	const values = useSelect( ( select ) => select( CORE_USER ).getUserInputSetting( slug ) || [] );
 	const [ other, setOther ] = useState( values.filter( ( value ) => ! options[ value ] )[ 0 ] || '' );
 	const { setUserInputSetting } = useDispatch( CORE_USER );
 	const inputRef = useRef();
+	const optionsRef = useRef();
 	const [ disabled, setDisabled ] = useState( false );
 
 	// Need to make sure that dependencies list always has the same number of elements.
 	const dependencies = values.concat( Array( max ) ).slice( 0, max );
 
+	useEffect( () => {
+		if ( ! isActive ) {
+			return;
+		}
+
+		const focusOption = ( element ) => {
+			if ( element ) {
+				setTimeout( () => {
+					element.focus();
+				}, 50 );
+			}
+		};
+
+		if ( optionsRef?.current ) {
```

Comment:
Same here, please, use the early return to reduce unnecessary nesting.

Reply:
👍